pkg/api/v1/router.go

```diff
@@ -60,4 +63,11 @@ func (r *Router) Routes(e *echo.Echo) {
 		r.addOriginRoutes(v1)
 		r.addPoolsRoutes(v1)
 	}
+
+	r.events.AddStream(&nats.StreamConfig{
+		Name: "lbapi",
```

Comment:
constant?

Reply:
I should make this configurable, but it's sort of in a grey area for my understanding of topics subscription vs consuming a stream. Would appreciate thoughts? 

In the integration env, a configurable stream means that there can be multiple operators consuming different streams. But then managers would see overlapping events possibly?